src/scoped.component.test.js

```diff
@@ -53,7 +57,7 @@ describe("<Scoped />", function() {
 
     expect(document.querySelectorAll(`style[type="text/css"]`)[0].innerHTML.trim()).toBe(
       `
-      [data-kremling="0"] .someRule, [data-kremling="0"].someRule {
```

Comment:
I removed `data-` from the dom attributes. See https://github.com/CanopyTax/kremling/issues/23